jinahub/indexers/searcher/NumpySearcher/tests/test_query.py

```diff
@@ -20,6 +23,30 @@ def test_query_vector(tmpdir):
     assert len(docs[0].matches[0].embedding) == 7
 
 
+@pytest.mark.parametrize(['metric', 'is_distance'],
+                         [('cosine', True), ('euclidean', True),
+                          ('cosine', False), ('euclidean', False)])
+def test_metric(tmpdir, metric, is_distance):
+    runtime = {
+        'workspace': str(tmpdir),
+        'name': 'searcher',
+        'pea_id': 0,
+        'replica_id': 0,
+    }
+
+    indexer = NumpySearcher(dump_path='tests/dump1', default_top_k=TOP_K, runtime_args=runtime, metric=metric,
```

Comment:
can u make this path relative to cur_dir?

Reply:
how about now?